server/api/project_apis.py

```diff
@@ -107,15 +115,18 @@ def get(self):
             search_dto.bbox = map(float, request.args.get('bbox').split(','))
             search_dto.input_srid = request.args.get('srid')
             search_dto.preferred_locale = request.environ.get('HTTP_ACCEPT_LANGUAGE')
+            search_dto.project_author = request.args.get('projectAuthor')
```

Comment:
Apologies, I thought this feature was for the user-profile not the project-dashboard.  So you were correct you can use a flag to limit by author, rather than provide the username.

In that case it's a lot simpler to just access the tm.authenticated_user param